scripts/provisioning/s3_prov_config.yaml

```diff
@@ -46,6 +46,33 @@ CONFIG:
   CONFSTORE_STORAGE_SET_SERVER_NODES_KEY: "cluster>cluster-id>storage_set[storage-set-count]>server_nodes"
   CONFSTORE_PUBLIC_FQDN_KEY: "server_node>machine-id>network>data>public_fqdn"
   CONFSTORE_PRIVATE_FQDN_KEY: "server_node>machine-id>network>data>private_fqdn"
+
+  CONFSTORE_S3SERVER_IP_ADDRESS: "cortx>software>s3>service>s3server_ip_address"
+  CONFSTORE_S3SERVER_PORT: "cortx>software>s3>service>s3server_port"
+  CONFSTORE_S3_MOTR_IP_ADDRESS: "cortx>software>s3>service>motr_ip_address"
+  CONFSTORE_S3_MOTR_PORT: "cortx>software>s3>service>motr_port"
```

Comment:
Call it something different - else it may potentially be misleading
MAybe REST_SERVER_FOR GB_ACTIVITIES